client/public/index.html

```diff
@@ -21,14 +21,29 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
 
-    <!-- Google Tag Manager -->
-    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
-      new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
-      j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
-      'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
-      })(window,document,'script','dataLayer','GTM-NMPT5JP');</script>
+    <!-- 
+      Google Tag Manager
+       
+      Note that we insert the GTM script tag from JS to ensure that
+      GTM only initializes itself once we've set the title tag.
+      For more details, see Page.tsx in this repository.
+    -->
+    <meta name="jf-gtm-id" content="GTM-NMPT5JP">
+    <script>
+      (function (window, dataLayer) {
+        window.dataLayer = window.dataLayer || [];
+        window.dataLayer.push({ "gtm.start": new Date().getTime(), event: "gtm.js" });
+      })(window, document, "script", "dataLayer", "GTM-NMPT5JP");
+    </script>
     <!-- End Google Tag Manager -->
-  
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag(){dataLayer.push(arguments);}
+      gtag('js', new Date());
+      gtag('config', 'UA-67069242-5');
```

Comment:
Wait shouldn't this just be removed?  This was Dan's old code from that commit two years ago, do we actually need it?  I think it just makes things more confusing...

Reply:
So, I think that's something we may need to check in with WholeWhale about— not sure if they're still utilizing our old Google Analytics account as a backup or what not

Reply:
Oh ok, I guess the thing I'm worried about is if GTM gets confused about seeing data layer events for two different accounts in it, using two different kinds of events?  (it seems like Dan's code adds a `js` event followed by a `config` event, while WW's just adds one `gtm.start` event.)

I think WW told me that they were fine with us still using our old analytics accounts but that it was entirely for our benefit rather than theirs... maybe ask them if it's ok for us to just remove the old stuff entirely?  I don't think it'd be a big deal if the legacy code used Google Analytics instead of an older version of Google Tag Manager--but the fact that it's the latter gives me all kinds of spielkus about GTM getting confused and not knowing which account to send events to...

Reply:
Fair point. Asking them about it now!